### ISSUE TITLE
Started Working Again

### ISSUE THREAD

**Issue by socialAPIS** [subins2000/InstaLV#5]
After a little bit of tweaking i've got it working again.

I will create a pull request when i will have free time.
![backtolife](https://user-images.githubusercontent.com/50284610/81556361-7c653980-938a-11ea-8fc0-72e516b5fd04.png)

**Comment by subins2000** [comment]
That'd be great. How did you get mgp25's library dependency tho ?

**Comment by JuaniSepulvedaM** [comment]
@socialAPIS can you help us to make it work??

**Comment by socialAPIS** [comment]
Yes i will Help but i really don't have time, i even don't have time to sleep. 

Middle this month or max end this month a pull request will come from me.

Im not using mgp's library.

**Comment by socialAPIS** [comment]
Sir after a long time I've found time and found how it works. I created a pull request, tested it and it works. But you can check it and tweak it a bit

**Comment by LitenBuzzTh** [comment]
> 
> 
> Sir after a long time I've found time and found how it works. I created a pull request, tested it and it works. But you can check it and tweak it a bit

thank you dear Sir. I will test your pull request tomorrow

**Comment by LitenBuzzTh** [comment]
> 
> 
> Sir after a long time I've found time and found how it works. I created a pull request, tested it and it works. But you can check it and tweak it a bit

unfortunately got this error :

C:\Users\USERRRR\Desktop\InstaLV>php C:\php\php-7.3.27\composer.phar install
Installing dependencies from lock file (including require-dev)
Verifying lock file contents can be installed on current platform.
Package operations: 11 installs, 0 updates, 0 removals
  - Downloading socialapis/instagram-api (V1.0.5)
 0/1 [>---------------------------]   0%    **Failed to download** socialapis/instagram-api from dist: The "https://api.github.com/repos/socialAPIS/Instagram-API/zipball/75bfd1fd2491f4e1e0a008b6cd020c40a00db81a" file could not be downloaded (HTTP/2 404 )
    Now trying to download from source
 1/1 [============================] 100%

  [RuntimeException]
  git was not found in your PATH, skipping source download


install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-suggest] [--no-dev] [--no-autoloader] [--no-scripts] [--no-progress] [--no-install] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--apcu-autoloader-prefix APCU-AUTOLOADER-PREFIX] [--ignore-platform-req IGNORE-PLATFORM-REQ] [--ignore-platform-reqs] [--] [<packages>]...

**Comment by socialAPIS** [comment]
I got screwed, i needed to delete the repo

**Comment by LitenBuzzTh** [comment]
> 
> 
> I got screwed, i needed to delete the repo

oh, I understant. is there any way that I can get this InstaLV working?

**Comment by socialAPIS** [comment]
I will send you my source code, what i have done, and my insta-api. But if you post it, give me credit.

How can i contact you?

**Comment by LitenBuzzTh** [comment]
> I will send you my source code, what i have done, and my insta-api. But if you post it, give me credit.
> 
> How can i contact you?
> 

This is my email address:
letsdev@hi2.in

Thanks
